src/openforms/formio/tests/test_datastructures.py

```diff
@@ -86,3 +88,62 @@ def test_initializing_with_dotted_paths_expands(self):
         }
 
         self.assertEqual(formio_data, expected)
+
+
+class FormioConfigurationWrapperTests(TestCase):
```

Comment:
I'm not sure if `FormioConfigurationWrapper` does DB operations, but if not we can switch to `SimpleTestCase`

Reply:
it's the `unittest.TestCase`, caught me off-guard too. I'm not sure what's best in those situations where you don't expect any django at all.

Reply:
Hum it's probably fine, looking at `SimpleTestCase` there's only a few assert helpers not related to Django, and the usage is pretty rare